helper/linux/writejob.h

```diff
@@ -63,4 +66,21 @@ public slots:
     QProcess *dd { nullptr };
 };
 
+
+template<class T>
+static std::tuple<std::unique_ptr<T[]>, T*, std::size_t> pageAlignedBuffer(std::size_t pages = 1024) {
+    static const std::size_t pagesize = getpagesize();
+    const std::size_t size = pages * pagesize;
+    std::size_t space = size + pagesize;
+    auto owner = std::make_unique<T[]>(space);
```

Comment:
There's QScopedPointer but it doesn't seem to support move semantics which is used below (in `return`).